docs/source/data_labeled.rst

```diff
@@ -57,37 +67,36 @@ train the first iteration of the active learning model.
 Fully labeled data
 ------------------
 
-.. note::
+.. tip::
 
 	Useful for Simulation and Exploration projects. Read more about :ref:`project_create:Project modes`.
 
-Fully labeled datasets are datasets with a review decision for each record in
+Fully labeled datasets are datasets with a labeling decision for each record in
 the dataset. Fully labeled datasets are useful for exploration or simulation
 purposes (see also :ref:`simulation_overview:What is a simulation?` and
-:ref:`project_create:Project modes`). See :ref:`data_labeled:Benchmark
-Datasets` for built-in, fully labeled datasets in ASReview LAB.
+:ref:`project_create:Project modes`). 
 
 
-Benchmark Datasets
-~~~~~~~~~~~~~~~~~~
+Benchmark data
+--------------
```

Comment:
```suggestion
Benchmark datasets
~~~~~~~~~~~~~~~~~~
```

I prefer to stick to benchmark datasets for now as the name aligns with the label used in ASReview itself. If we disagree on this label in the frontend, we might want to change it there as well. 

I think we can also keep `~` instead of `-`, as this is a subsection of Fully labeled data. This keeps the side menu clear for the reader.